packages/vue/src/components/alias-control.vue

```diff
@@ -1,3 +1,65 @@
+<script setup lang="ts">
+import { computed, ComputedRef, onMounted } from 'vue';
+import {
+  ActorContextWithForms,
+  authInputAttributes,
+  countryDialCodes,
+  getActorContext,
+  LoginMechanism,
+} from '@aws-amplify/ui';
+
+import { useAuth } from '../composables/useAuth';
+
+interface PropsInterface {
+  label: string;
+  name: string;
+  placeholder?: string;
+  autocomplete?: string;
+}
+
+const { label, name, placeholder, autocomplete } = withDefaults(
+  defineProps<PropsInterface>(),
+  {
+    label: 'Username',
+    name: 'username',
+    placeholder: '',
+    autocomplete: '',
+  }
+);
+const random = Math.floor(Math.random() * 999999);
+const randomPhone = Math.floor(Math.random() * 999999);
+
+const { state, send } = useAuth();
+const {
+  value: { context },
+} = state;
+
+//computed
+const inputAttributes = computed(() => authInputAttributes);
+const actorContext: ComputedRef<ActorContextWithForms> = computed(() =>
+  getActorContext(state.value)
+);
+
+const defaultDialCode = actorContext.value.country_code;
+
+const dialCodes = computed(() => countryDialCodes);
+
+onMounted(() => {
+  if (inputAttributes.value[name as LoginMechanism].type === 'tel') {
+    send({
+      type: 'CHANGE',
+      data: { name: 'country_code', value: defaultDialCode },
+    });
+  }
+});
+
+const inferAutocomplete = computed((): string => {
+  return (
+    autocomplete || authInputAttributes[name as LoginMechanism]?.placeholder
```

Comment:
```suggestion
    autocomplete || authInputAttributes[name as LoginMechanism]?.autocomplete
```

This is my bad! I see I made the same typo in Angular as well... 😓  I'll fix the Angular one soon.

Reply:
Ha! Thanks, that's what I get for copying and pasting without checking. I'll fix it